src/ln/channelmanager.rs

```diff
@@ -2138,16 +2131,7 @@ impl ChannelManager {
 					try_chan_entry!(self, chan.get_mut().channel_reestablish(msg), channel_state, chan);
```

Comment:
L2130, now useless order mutability

Reply:
Backed this change out in a new version of the stack, turns out removing this isnt correct.